README.md

```diff
@@ -27,12 +63,13 @@ The governing flow for how tests are executed stem from the `.evergreen/config.y
 
 -   `name` -- This should be in the format test-{pipeline}-{language}-{os}
 -   `display_name` -- This can be named however you see fit. Ensure it is easy to understand. See `.evergreen/config.yml` for examples
--   [`expansions`](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Configuration-Files/#expansions) -- These are how we store buildvariant specific variables. Additionally, for expansions that need to be maintained as secrets, please update those in t[he evergreen project settings](https://spruce.mongodb.com/project/ai-ml-pipeline-testing/settings/variables) using [variables](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-and-Distro-Settings#variables). Some common expansions this takes are:
+-   [`expansions`](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Configuration-Files/#expansions) -- These are how we store buildvariant specific variables. Additionally, for expansions that need to be maintained as secrets, please update those in [the evergreen project settings](https://spruce.mongodb.com/project/ai-ml-pipeline-testing/settings/variables) using [variables](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-and-Distro-Settings#variables). Some common expansions needed are:
     -   `DIR` -- The subdirectory where the tasks will run
     -   `REPO_NAME` -- The name of the repository that will get cloned
     -   `CLONE_URL` -- The github url to clone into the specified DIR
+    -   `DATABASE` -- The database atlas cli will load your index configs (optional)
 
--   `run_on` -- Specified platform to run on. `ubuntu2204-small` is a fine default
+-   `run_on` -- Specified platform to run on. `rhel87-small` is a fine default. Any other distro may fail atlas cli setup.
 -   `tasks` -- Tasks to run. See below for more details
 -   `cron` -- The tests are run via a cron job on a weekly cadence. However, this can be augmented by setting a different cadence. Cron jobs can be scheduled using [cron syntax](https://crontab.guru/#0_0_*_*_0)
```

Comment:
```suggestion
-   `cron` -- The tests are run via a cron job on a weekly cadence. This can be modified by setting a different cadence. Cron jobs can be scheduled using [cron syntax](https://crontab.guru/#0_0_*_*_0)
```

I thought we were planning to run these tests nightly?

Reply:
Yep. We did. Went ahead and edited the change to reflect that. (It's running nightly now)